distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/PaxExamRuleTest.java

```diff
@@ -175,9 +176,13 @@ public void failingBeforeExam() {
         JUnitCore core = new JUnitCore();
         Result result = core.run(FailingBeforeExamTest.class);
 
-        assertThat(result.getFailures()).extracting("message")
-                .containsOnly(EXPECTED_BEFORE_EXAM_ERROR_MESSAGE,
-                        PaxExamRule.EXAM_SETUP_FAILED_MESSAGE);
+        assertThat(result.getFailures()
```

Comment:
This formatting is terrible. Maybe extract the two boolean terms into variables so it becomes more readable?